management/lifecycle.py

```diff
@@ -1096,7 +1104,7 @@ def main():
         # take highest one and set priority downstream
         # if no priority label(s), DO NOT set priority at all for this PR
         if args.set_priority:
-            labels =  set(map(lambda x: x.name, pr.labels))
+            labels =  _get_labels(pr)
```

Comment:
Since this gets &'ed in the next line, we need to ensure that this is a set
```suggestion
            labels =  set(_get_labels(pr))
```

Reply:
Instead of this change I altered `_get_labels` to return `list(set(label.name for label in pr.get_labels()))`

Reply:
I don't think that'll help - the `&` operator isn't defined for lists

Reply:
You may need to look at this in the file diff tab to see what I mean, since the line with the `&` is after the line where the change is proposed